docs/extensions/hooks.md

```diff
@@ -191,8 +191,12 @@ export default ({ embed }, { env }) => {
 
 ### Filter Events
 
-| Name                           | Payload                              | Meta                                        |
+| Event                          | Payload                              | Meta                                        |
 | ------------------------------ | ------------------------------------ | ------------------------------------------- |
+| `websocket.message`            | The message send over the websocket  |                                             |
+| `fields.create`                | The raw field you created            | `collection`                                |
+| `fields.update`                | The raw field you updated            | `keys`, `collection`                        |
+| `fields.delete`                | The raw field you deleted            | `collection`                                |
```

Comment:
Those events are already in the table as part of `<system-collection>.create`, `<system-collection>.update` and `<system-collection>.delete` since `fields` is a system collection and is mentioned in the "System Collections" tip box below.